examples/annotation_import/dicom.ipynb

```diff
@@ -0,0 +1,534 @@
+        "\n",
+        "For example, when we create the line annotation above, we provided the `name` as `line_dicom`. Now, when we setup our ontology, we must ensure that the name of my line tool is also `line_dicom`. The same alignment must hold true for the other tools and classifications we create in our ontology.\n",
+        "\n",
+        "\n",
+        "[Documentation for reference ](https://docs.labelbox.com/reference/import-text-annotations)"
+      ],
+      "cell_type": "markdown"
+    },
+    {
+      "metadata": {},
+      "source": [
+        "ontology_builder = lb.OntologyBuilder(\n",
+        "    tools=[\n",
+        "        lb.Tool(tool=lb.Tool.Type.RASTER_SEGMENTATION, name=\"segmentation_mask_dicom\"),\n",
+        "        lb.Tool(tool=lb.Tool.Type.LINE, name=\"line_dicom\"),\n",
+        "    ]\n",
+        ")\n",
+        "\n",
+        "ontology = client.create_ontology(\"Ontology DICOM Annotations\", ontology_builder.asdict(), media_type=lb.MediaType.Dicom)"
+      ],
+      "cell_type": "code",
+      "outputs": [],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "### Step 3: Create a labeling project \n",
+        "Connect the ontology to the labeling project."
+      ],
+      "cell_type": "markdown"
+    },
+    {
+      "metadata": {},
+      "source": [
+        "# Project defaults to batch mode with benchmark quality settings if this argument is not provided\n",
+        "# Queue mode will be deprecated once dataset mode is deprecated\n",
+        "\n",
+        "project = client.create_project(name=\"dicom_project_demo\", media_type=lb.MediaType.Dicom)\n",
+        "\n",
+        "## connect ontology to your project\n",
+        "project.setup_editor(ontology)"
+      ],
+      "cell_type": "code",
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "Default createProject behavior will soon be adjusted to prefer batch projects. Pass in `queue_mode` parameter explicitly to opt-out for the time being.\n"
+          ]
+        }
+      ],
+      "execution_count": null
+    },
+    {
+      "metadata": {},
+      "source": [
+        "### Step 4: Send a batch of data rows to the project"
+      ],
+      "cell_type": "markdown"
+    },
+    {
+      "metadata": {},
+      "source": [
+        "# Create batches\n",
+        "\n",
+        "# Create a batch to send to your MAL project\n",
+        "batch = project.create_batch(\n",
+        "  \"first-batch-dicom-demo\", # Each batch in a project must have a unique name\n",
+        "  global_keys=[global_key], # A paginated collection of data row objects\n",
```

Comment:
One more thing, let's add a comment here that says "# Paginated collection of data row objects, list of data row ids or global keys"

Reply:
This is not a blocker - so PR approved

Reply:
Good call, updated!